openmetadata-ui/src/main/resources/ui/src/components/Glossary/GlossaryTerms/tabs/AssetsTabs.component.tsx

```diff
@@ -413,7 +413,11 @@ const AssetsTabs = forwardRef(
             type={ERROR_PLACEHOLDER_TYPE.FILTER}
           />
         );
-      } else if (noDataPlaceholder || searchValue || !permissions.Create) {
+      } else if (
+        isObject(noDataPlaceholder) ||
```

Comment:
Hello @Ashish8689 , can you explain why did we need this change?

Reply:
the noDataPlaceholder can be an object or string. and in this particular condition, we are using value only from object. So to further use the noDataPlaceholder as a string have used this condition.


<img width="429" alt="image" src="https://github.com/open-metadata/OpenMetadata/assets/66266464/1a90b2ea-2158-49c5-99c6-5c4b11c070f5">